VaticanBAV/et/BAVet153.xml

```diff
@@ -0,0 +1,221 @@
+                            <locus from="1r" to="2v"/>
+                            <locus from="54v" to="56v"/>
+                            <dimensions type="outer" unit="mm">
+                              <height>158</height>
+                              <width>103</width>
+                            </dimensions>
+                            </extent>
+                          
+                            <collation>
+                     <note>The manuscript is composed of one protective bifolio, five quinions, and one binion. The indication of their sequence
+                     given here is a conjecture based on the somewhat vague description of the catalogue.</note>
+                              <list>
+                                <item xml:id="q1" n="A">
+                                  <dim unit="leaf">2</dim>
+                                  <locus from="1" to="2"/>
+                                </item>
+                                <item xml:id="q2" n="1">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="3" to="12"/>
+                                </item>
+                                <item xml:id="q3" n="2">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="13" to="22"/>
+                                </item>
+                                <item xml:id="q4" n="3">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="23" to="32"/>
+                                </item>
+                                <item xml:id="q5" n="4">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="33" to="42"/>
+                                </item>
+                                <item xml:id="q6" n="5">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="43" to="52"/>
+                                </item>
+                                <item xml:id="q7" n="6">
+                                  <dim unit="leaf">4</dim>
+                                  <locus from="53" to="56"/>
+                                </item>
+                              </list>
+                            </collation>
+                          </supportDesc>
+                        
+                        <layoutDesc>
+                          <layout columns="2" writtenLines="17"/>
+                        </layoutDesc>
+                        </objectDesc>
+                      
+                        <handDesc>
+                          <handNote script="Ethiopic" xml:id="h1">
+                            <desc>Neatly written
+                              (the catalogue description ‘nitide exaratum’ refers to the codex as a whole, 
+                              but might be assumed to mainly describe the hand).</desc>
```

Comment:
The words "nitide exaratum" are most probably referring to the scribal hand, rather unlikely to the volume as a whole (here and in similar cases).

Reply:
I agree, but it is still an interpretation...anyway, I can delete the explanation